.github/semantic.yml

```diff
@@ -102,6 +102,7 @@ scopes:
   - sematext provider
 
   # sources
+  - apache_metrics source
```

Comment:
Was this auto-generated from `.github/semantic.yml.erb`? (and if so, how did you do that 👀).

I need to add one for the `remap` transform as well, but I haven't looked into where the actual `.metadata` data comes from when it generates this file.

Reply:
I actually missed that this was a generated file. I'm actually not sure how to regenerate it. I assume @binarylogic knows? Does this repo need a `generate.rb` like `vector-website` has?